apps/minifront/src/components/swap/swap-loader.ts

```diff
@@ -1,29 +1,22 @@
 import { LoaderFunction } from 'react-router-dom';
-import { useStore } from '../../../state';
+import { useStore } from '../../state';
 import { throwIfPraxNotConnectedTimeout } from '@penumbra-zone/client/prax';
-import {
-  BalancesResponse,
-  SwapRecord,
-} from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/view/v1/view_pb';
+import { SwapRecord } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/view/v1/view_pb';
 import { Metadata } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/asset/v1/asset_pb';
-import { fetchUnclaimedSwaps } from '../../../fetchers/unclaimed-swaps';
-import { viewClient } from '../../../clients';
+import { fetchUnclaimedSwaps } from '../../fetchers/unclaimed-swaps';
+import { viewClient } from '../../clients';
 import { getSwapAsset1, getSwapAsset2 } from '@penumbra-zone/getters/swap-record';
 import { uint8ArrayToBase64 } from '@penumbra-zone/types/base64';
-import { getSwappableBalancesResponses, isSwappable } from '../helpers';
-import { getAllAssets } from '../../../fetchers/assets';
+import { getSwappableBalancesResponses, isSwappable } from './helpers';
+import { getAllAssets } from '../../fetchers/assets';
 
 export interface UnclaimedSwapsWithMetadata {
   swap: SwapRecord;
   asset1: Metadata;
   asset2: Metadata;
 }
 
-export interface SwapLoaderResponse {
-  assetBalances: BalancesResponse[];
-  unclaimedSwaps: UnclaimedSwapsWithMetadata[];
-  swappableAssets: Metadata[];
-}
+export type SwapLoaderResponse = UnclaimedSwapsWithMetadata[];
```

Comment:
`assetBalances` and `swappableAssets` are moved into Zustand state (although they are loaded by the loader, and then stored in Zustand). So we only return unclaimed swaps here. Eventually, we'll move that into Zustand as well, but that's out of scope for this PR.